generics/services/kendra.js

```diff
@@ -0,0 +1,125 @@
+/**
+ * name : kendra.js
+ * author : Aman Jung Karki
+ * Date : 26-June-2020
+ * Description : All kendra service related information.
+ */
+
+//dependencies
+
+const request = require('request');
+const fs = require("fs");
+
+/**
+  * Get downloadable file.
+  * @function
+  * @name getDownloadableUrl
+  * @param {Object} bodyData - body data.
+  * @returns {Array} Downloadable file.
+*/
+
+const getDownloadableUrl = function (bodyData) {
+
+    let fileDownloadUrl = process.env.KENDRA_APPLICATION_ENDPOINT; 
+    
+    if ( process.env.CLOUD_STORAGE === "GC" ) {
+        fileDownloadUrl = fileDownloadUrl + messageConstants.endpoints.DOWNLOADABLE_GCP_URL;
+        bodyData.bucketName = process.env.GCP_BUCKET_NAME;
+    } else if (process.env.CLOUD_STORAGE === "AWS" ) {
+        fileDownloadUrl = fileDownloadUrl + messageConstants.endpoints.DOWNLOADABLE_AWS_URL;
+        bodyData.bucketName = process.env.AWS_BUCKET_NAME;
+    } else {
+        fileDownloadUrl = fileDownloadUrl + messageConstants.endpoints.DOWNLOADABLE_AZURE_URL;
+        bodyData.bucketName = process.env.AZURE_STORAGE_CONTAINER;
+    }
+
+    return new Promise((resolve, reject) => {
+        try {
+
+            const kendraCallBack = function (err, response) {
+                if (err) {
+                    return reject({
+                        status : httpStatusCode.bad_request.status,
+                        message : messageConstants.apiResponses.KENDRA_SERVICE_DOWN
+                    })
+                } else {
+                    let downloadableImage = response.body;
+                    return resolve(downloadableImage);
+                }
+            }
+
+            request.post(fileDownloadUrl, {
+                headers: {
+                    "internal-access-token": process.env.INTERNAL_ACCESS_TOKEN
+                },
+                json : bodyData
+            }, kendraCallBack);
+
+        } catch (error) {
+            return reject(error);
+        }
+    })
+
+}
+
+/**
+  * Upload urls.
+  * @function
+  * @name upload
+  * @param {Object} bodyData - body data.
+  * @returns {Array} upload file.
+*/
+
+const upload = function (file,filePath) {
+
+    let fileUploadUrl = process.env.KENDRA_APPLICATION_ENDPOINT; 
```

Comment:
@aman-tunerlabs - Line 75-87 ca be one common function for upload and download function